src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7564,9 +7567,13 @@ public static partial class File
         public static System.IO.FileStream Open(string path, System.IO.FileMode mode) { throw null; }
         public static System.IO.FileStream Open(string path, System.IO.FileMode mode, System.IO.FileAccess access) { throw null; }
         public static System.IO.FileStream Open(string path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share) { throw null; }
+        public static System.IO.FileStream Open(string path, System.IO.FileMode mode, System.IO.FileStreamOptions options) { throw null; }
```

Comment:
https://github.com/dotnet/runtime/issues/24698#issuecomment-851386325

```suggestion
        public static System.IO.FileStream Open(string path, System.IO.FileStreamOptions options) { throw null; }
```